homework_postgres/PostgreSQL-Students.ipynb

```diff
@@ -40,9 +40,22 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    Customers_Cnt= psql.read_sql("""SELECT DISTINCT name, COUNT (*) 
</pre><p>The code tutorial make some mistake here, groupy by does not need distinct again cause groupy by already group duplicate country</p><p>Redundant Distinct cause groupy already group the same name of customer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/haicheviet/dataeng-course-1/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>